test/e2e/README.md

```diff
@@ -1,6 +1,59 @@
-# End-to-End Tests
+# Kubestellar end to end testing
+
```

Comment:
This is the introduction to E2E testing. It should start with an explanation of the basic lay of the land. Something like the following. It should explain that this filesystem branch and the run-test.sh script here support the following test matrix.

- Run either one of two scenarios
- Either create three new `kind` clusters or use three existing OpenShift clusters
- Either test the local repo copy or last release before this version of the repo

with the exception that OCP can only be used with a release.